test/mint.js

```diff
@@ -1,81 +1,144 @@
-const chai = require("chai");
-const { expect } = chai;
-const { ethers } = require("hardhat");
-const chaiAsPromised = require("chai-as-promised");
+const chai = require('chai')
+const { expect } = chai
+const { ethers } = require('hardhat')
+const chaiAsPromised = require('chai-as-promised')
 
-chai.use(chaiAsPromised);
-describe("PodCastNFT free mint", function () {
-  let Contract, contract;
+chai.use(chaiAsPromised)
+describe('PodCastNFT', function () {
+  let Contract, contract
+  let owner, alice, bob
 
   beforeEach(async function () {
-    Contract = await ethers.getContractFactory("PodCastNFT");
-  });
+    Contract = await ethers.getContractFactory('PodCastNFT')
+    ;[owner, alice, bob] = await ethers.getSigners()
+    contract = await Contract.deploy()
+    await contract.deployed()
+  })
 
-  it("can mint for admin", async function () {
-    const [owner, user1] = await ethers.getSigners();
-    contract = await Contract.deploy([owner.address], false);
-    await contract.deployed();
+  describe('getRoles', () => {
+    it('revert without nft', async () => {
+      await expect(contract.getRoles(alice.address)).to.be.revertedWith(
+        'wallet must have membership nft'
+      )
+    })
+    it('can retrieve roles', async () => {
+      const mintTx = await contract.mint(
+        'https://example.com/podcast.png',
+        ['Podcast Contributor', 'MEMBER'],
```

Comment:
Probably standardise the role format to either CAPITAL or Camel Case to keep it consistent

Reply:
yeah, that's the things we need to decide.
maybe camel case is better cuz we will use as metadata of NFT as well